src/enums/warning-types.ts

```diff
@@ -0,0 +1,7 @@
+export enum WarningTypes {
```

Comment:
we have file naming and Enum types by appending letter E at the end.

Reply:
I did just missed it here thanks